src/adapter/src/coord/catalog_implications.rs

```diff
@@ -301,13 +303,14 @@ impl Coordinator {
                     }
                 }
                 CatalogImplication::Sink(CatalogImplicationKind::Added(sink)) => {
-                    tracing::debug!(?sink, "not handling AddSink in here yet");
+                    sinks_to_create.push((catalog_id, sink));
                 }
                 CatalogImplication::Sink(CatalogImplicationKind::Altered {
-                    prev: prev_sink,
-                    new: new_sink,
+                    prev: _prev_sink,
+                    new: _new_sink,
                 }) => {
-                    tracing::debug!(?prev_sink, ?new_sink, "not handling AlterSink in here yet");
+                    // Sink alterations are not yet supported - they would require
+                    // re-creating the storage export with new parameters.
```

Comment:
Hm, but we do support `ALTER SINK ... SET FROM`. Am I missing something?

Reply:
I'd be happy with modifying the comment to make it clear that "not yet supported" is in the context of the implications, not in the context of Materialize!